bin/upload-video

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env ruby
+
+# This class processes and uploads a video to s3,
+# making it available for viewing through Code.org's fallback video player.
+
+# Usage:
+# upload-video [youtube id] [video filename]
+#
+# Dependency requirements:
+#
+# ffmpeg (`brew install ffmpeg` / `apt-get install ffmpeg`) - for transcoding manually-provided video files
+# s3_access_key_id and s3_secret_access_key defined (e.g. in `locals.yml`)
+
+# To use this script to process all YouTube video tags in custom levels, run this command from the root folder (between =begin and =end):
```

Comment:
Would it make sense to check in the script for processing all YouTube video tags in custom levels?  (That can happen in another pull request)